### DISCUSSION TITLE
Modal background color

### DISCUSSION THREAD

**Discussion by peridotml** [mantinedev/mantine#5717] [Q&A]
I am unable to change the modal background color to black. Would appreciate some help! Thank you!

**Comment by yverby** [comment]
You can override `--mantine-color-body` css variable used for modal content elements as `background-color`
```jsx
<Modal styles={{ content: { '--mantine-color-body': 'black' } }}>
  ...
</Modal>
```
You can also use native css, just put the same property in your `content` class
